storage/innobase/xtrabackup/utils/percona-xtrabackup.spec

```diff
@@ -66,27 +66,20 @@ export CXX=${CXX-"g++"}
 export CFLAGS=${CFLAGS:-}
 export CXXFLAGS=${CXXFLAGS:-}
 #
-cmake -DBUILD_CONFIG=xtrabackup_release -DCMAKE_INSTALL_PREFIX=%{_prefix} -DMYSQL_UNIX_ADDR="%{mysqldatadir}/mysql.sock" \
-  -DWITH_SSL=system -DINSTALL_MYSQLTESTDIR=%{_datadir}/percona-xtrabackup-test -DINSTALL_MANDIR=%{_mandir} .
+cmake -DBUILD_CONFIG=xtrabackup_release -DCMAKE_INSTALL_PREFIX=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor} -DMYSQL_UNIX_ADDR="%{mysqldatadir}/mysql.sock" \
+  -DWITH_SSL=system -DINSTALL_MYSQLTESTDIR=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}/percona-xtrabackup-test -DINSTALL_MANDIR=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor} .
 #
 make %{?_smp_mflags}
 #
 %endif
 #
 %install
 rm -rf $RPM_BUILD_ROOT
-install -d $RPM_BUILD_ROOT/opt
-install -d $RPM_BUILD_ROOT/opt/percona-xtrabackup
-install -d $RPM_BUILD_ROOT/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}
-make install DESTDIR=$RPM_BUILD_ROOT/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}
-install -d $RPM_BUILD_ROOT/%{_datadir}
-install -d $RPM_BUILD_ROOT/%{_datadir}/percona-xtrabackup-test
-mv $RPM_BUILD_ROOT/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}/usr/share/percona-xtrabackup-test $RPM_BUILD_ROOT/usr/share/percona-xtrabackup-test
-
+make install DESTDIR=$RPM_BUILD_ROOT
 
 %post
-BIN_PATH=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}/usr/bin
-DOC_PATH=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}/usr/share/man/man1
+BIN_PATH=/opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}/bin
```

Comment:
it's better to define prefix globally as /opt/percona-xtrabackup/%{xb_version_major}.%{xb_version_minor}